app/src/main/java/com/greenfox/kalendaryo/SelectCalendarActivity.java

```diff
@@ -61,6 +61,8 @@ protected void onCreate(Bundle savedInstanceState) {
                 new DividerItemDecoration(recKal.getContext(),
                         recyclerLayoutManager.getOrientation());
         recKal.addItemDecoration(dividerItemDecoration);
+        String customName = getIntent().getStringExtra("customName");
```

Comment:
should use the pre defined const from `CustomNameActivity.java` like `CustomNameActivity.CALENDAR_NAME`